src/core/graphics/brushes/SolidBrush.js

```diff
@@ -0,0 +1,70 @@
+var Brush = require('./Brush');
+
+/**
+ * A Simple single-color brush.
+ *
+ * @class
+ * @extends Brush
+ * @param color {Number} The color of the brush
+ * @param alpha {Number} The alpha of the brush
+ * @memberof PIXI
+ */
+function SolidBrush(color, alpha)
+{
+    Brush.call(this, color, alpha);
+
+    /**
+     * @member {String} the value that can be assigned to a context.fillStyle of StrokeStyle
+     * @private
+     */
+    this._canvasBrush = null;
+}
+
+SolidBrush.prototype = Object.create(Brush.prototype);
+SolidBrush.prototype.constructor = SolidBrush;
+
+
+/*
+ * Generates _canvasBrush based on the tint color
+ *
+ * @param tint {Number} the tint color
+ * @param tintR {Number} the R component of the tint color
+ * @param tintG {Number} the G component of the tint color
+ * @param tintB {Number} the B component of the tint color
+ *
+ */
+SolidBrush.prototype.setTint = function (tint, tintR, tintG, tintB)
```

Comment:
This seems wrong, and definitely does not follow the pattern in the rest of pixi.

Most likely this should just be a tint color param that uses `utils.hex2rgb()`. Having both params doesn't really make sense.